tools/gc_derivatization/.shed.yml

```diff
@@ -0,0 +1,16 @@
+
+name: gc_derivatization
+owner: recetox
+remote_repository_url: "https://github.com/RECETOX/galaxytools/tree/master/tools/gc_derivatization"
+homepage_url: "https://github.com/RECETOX/galaxytools"
+categories:
+  - Metabolomics
+description: Performs GC derivatization.
+long_description: |
+    The GC-derivatization tool converts carbonyl groups to C═N–OCH3 (MeOX) and transforms acidic protons into -Si(CH3)3 (TMS). Key functionalities include checking for 
+    specific groups, removing derivatization groups, and adding derivatization groups to molecules.
+include:
```

Comment:
Try to get rid of include, its hard to maintain such a list.

Reply:
Done [88ae07a](https://github.com/RECETOX/galaxytools/pull/503/commits/88ae07aff959d895e5c28b5ea2b11c7b30902d95)